src/main/java/com/meetup/server/auth/dto/response/JwtUserDetails.java

```diff
@@ -0,0 +1,53 @@
+package com.meetup.server.auth.dto.response;
+
+import io.jsonwebtoken.Claims;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+import java.util.Collections;
+
+public record JwtUserDetails(
+        Long userId
```

Comment:
```suggestion
        Long memberId
```

User랑 Member가 혼용되고 있는 거 같아요! Member로 통일하면 좋을 거 같습니다!